src/dialogs/AboutDialog.cpp

```diff
@@ -48,7 +48,7 @@ AboutDialog::AboutDialog(QWidget *parent) :
                         + tr("This Software is released under the GNU General Public License v3.0")
                         + "<h2>" + tr("Authors") + "</h2>"
                         + tr("Cutter is developed by the community and maintained by its core and development teams.<br/>")
-                        + tr("Check our <a href='https://github.com/radareorg/cutter/graphs/contributors'>contributors page</a> for the full list of contributors."));
+                        + tr("Check our <a href='https://github.com/rizinorg/cutter/graphs/contributors'>contributors page</a> for the full list of contributors."));
```

Comment:
thats wrong, also, i have started the rebranding to the original name, no more cutter mentions in here